lms/static/js/views/fields.js

```diff
@@ -601,10 +601,25 @@
 
             updateCharCount: function() {
                 var curCharCount;
+                var remainingCharCount;
+                var $charCount = $('.u-field-footer .current-char-count');
                 // Update character count for textarea
                 if (this.options.maxCharacters) {
                     curCharCount = $('#u-field-textarea-' + this.options.valueAttribute).val().length;
-                    $('.u-field-footer .current-char-count').text(curCharCount);
+                    remainingCharCount = this.options.maxCharacters - curCharCount;
+                    if (remainingCharCount < 70 && remainingCharCount > 60) {
+                        $charCount.attr('aria-live', 'polite');
+                        $charCount.attr('aria-atomic', 'true');
+                    }
+                    else if (remainingCharCount < 60 && remainingCharCount > 20) {
+                        $charCount.attr('aria-atomic', 'false');
+                    }
+                    else if (remainingCharCount < 20) {
+                        $charCount.attr('aria-live', 'assertive');
+                        $charCount.attr('aria-atomic', 'true');
+                    }
```

Comment:
If this was reordered to the following I think it would be easier to read;
```
if (remainingCharCount < 20) {....
} else if (remainingCharCount < 60) {...
} else if (remainingCharCount < 70) {...
}
```